### PR TITLE
Adding App Engine pull task queue sample

### PR DESCRIPTION
**Pull request by jsimonweb**

### CODE REVIEW

File: appengine/taskqueue/pull/eclipse-launch-profiles/DevAppServer.launch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (lesv) [inline appengine/taskqueue/pull/eclipse-launch-profiles/DevAppServer.launch]: Don't include

File: appengine/taskqueue/pull/eclipse-launch-profiles/UpdateApplication.launch [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (lesv) [inline appengine/taskqueue/pull/eclipse-launch-profiles/UpdateApplication.launch]: don't include.

File: appengine/taskqueue/pull/nbactions.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+#set( $symbol_pound = '#' )
```
Comment (lesv) [inline appengine/taskqueue/pull/nbactions.xml]: Argh - Ludo likes these, but we are publishing w/o them.

File: appengine/taskqueue/pull/pom.xml [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>war</packaging>
+    <version>1.0-SNAPSHOT</version>
+    <groupId>com.example.taskqueue</groupId>
+    <artifactId>taskqueue</artifactId>
+
+    <properties>
```
Comment (lesv) [inline appengine/taskqueue/pull/pom.xml]: Please reference the parent pom.xml, where most of this comes from.

File: appengine/taskqueue/pull/pom.xml [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+            <version>${appengine.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <!-- for hot reload of the web application-->
+        <outputDirectory>${project.build.directory}/${project.build.finalName}/WEB-INF/classes</outputDirectory>
+        <plugins>
+            <plugin>
```
Comment (lesv) [inline appengine/taskqueue/pull/pom.xml]: versions is or should be in the parent.

File: appengine/taskqueue/pull/pom.xml [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>display-dependency-updates</goal>
+                            <goal>display-plugin-updates</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```
Comment (lesv) [inline appengine/taskqueue/pull/pom.xml]: not required as it's set in the parent.

File: appengine/taskqueue/pull/pom.xml [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+                <version>3.1</version>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.7</source>
+                    <target>1.7</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```
Comment (lesv) [inline appengine/taskqueue/pull/pom.xml]: Not really required.

File: appengine/taskqueue/pull/src/main/webapp/WEB-INF/appengine-web.xml [hunk 8]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (lesv) [inline appengine/taskqueue/pull/src/main/webapp/WEB-INF/appengine-web.xml]: License here.

File: appengine/taskqueue/pull/src/main/webapp/WEB-INF/logging.properties [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# A default java.util.logging configuration.
```
Comment (lesv) [inline appengine/taskqueue/pull/src/main/webapp/WEB-INF/logging.properties]: License Here

File: appengine/taskqueue/pull/src/main/webapp/WEB-INF/web.xml [hunk 10]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lesv) [inline appengine/taskqueue/pull/src/main/webapp/WEB-INF/web.xml]: License here

File: appengine/taskqueue/pull/src/main/webapp/tasks.jsp [hunk 11]
```diff
@@ -0,0 +1,36 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
```
Comment (lesv) [inline appengine/taskqueue/pull/src/main/webapp/tasks.jsp]: License can go here, also a `<!DOCTYPE html>`

**Comment by lesv** [comment]
Please make the requested changes, then Ok to merge.

**Comment by lesv** [comment]
(Actually wait 'til travis passes)

**Comment by jsimonweb** [comment]
Made requested changes. Now waiting on travis.

File: taskqueue/pull/TaskQueueSample.launch [hunk 12]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (lesv) [inline taskqueue/pull/TaskQueueSample.launch]: You might wish to skip this file.

**Comment by lesv** [comment]
passed - one file you might wish to remove

**Comment by jsimonweb** [comment]
File removed. Preparing to merge..

**Pull request merged**